### DISCUSSION TITLE
where i can host my code in github

### DISCUSSION THREAD

**Discussion by sulaimanisyaku** [community/community#44452] [New to GitHub]
### Select Topic Area

General

### Body

where i can host my code in github

**Comment by skadomsky** [comment]
In your repository. To create a repo you need to click on the "+" button on the right above corner and select "New Repository". Since you done everything you need to set up a repo, you can add your code using git tool or uploading files.
Here are offical documentations by GitHub which you can take as a reference: https://docs.github.com/en/get-started/quickstart/create-a-repo
